### DISCUSSION TITLE
Set PWM from Berry

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#21063] [Q&A]
How to set pwm value from berry script?

Has set one output to PWM1.
i want to keep the slider in webgui.

```
light.get()
{'bri': 255, 'channels': [255], 'power': true}
light.set({"channels": [32]})
{'bri': 0, 'channels': [0], 'power': true}
```

**Comment by wjohn007** [comment]
the complete set of console commands is also available for berry using tasmota.cmd.

e.g.
` tasmota.cmd("pwm 512")`
